### ISSUE TITLE
Notes [design review call] (don't close)

### ISSUE THREAD

**Issue by marcoambrosini** [nextcloud/deck#6622]
**Important**: ensure that when it applies, changes are also reflected on the mobile apps

#### First load and dummy data
- [ ] First load of the app: load personal view instead of upcoming cards
- [ ] Dummy data: come up with more relevant cards
  - [ ] Tutorial tasks as one idea like other tools do it

#### Search and filtering
- [ ] Search should go to the top of the navigation
  - [ ] We can filter the navigation
  - [ ] Then we don't need the headings anymore
    - [ ] We could have quick filters that apply to the list 
  - [ ] We should search for boards and cards like talk does
  - [ ] Remove the global serach in the bottom content area
- [ ] Epic: Search and filtering needs some bigger design discussion
- [ ] Smaller things filter menu
  - [ ] Label filter button in the top right if space is available
  - [ ] Headings are a bit too big
  - [ ] Tag list might be better to switch to a multiselect or have a filter -> bigger discussion as ewll

#### Settings
- [ ] Settings
  - [ ] Move to a settings dialog
  - [ ] Keyboard shortcuts already has a dialog
- [ ] Add board input. can stay for now
- [ ] Skipped discussing modal due to https://github.com/nextcloud/deck/issues/2702

- [ ] Big button to the right of the last list to create new one (still keep an ) 
- [ ]Button to create new card: same behaviour as trello, screenshots below

## ✅ Done
<details>
<summary> Before starting a new design review, move all completed items into this bucket</summary>

Placeholder 

</details>

<details>
<summary> 🗓️ Reviews done:</summary>
2024/12/17
</details>

cc @juliusknorr
